app/jobs/bulk_update/add_trainees/import_rows_job.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module BulkUpdate
+  module AddTrainees
+    class ImportRowsJob < ApplicationJob
+      queue_as :bulk_update
+
+      def perform(id:)
+        return unless FeatureService.enabled?(:bulk_add_trainees)
+
+        ImportRows.call(id:)
```

Comment:
Since [BulkUpdate:: BulkAddTraineesUploadForm:27](https://github.com/DFE-Digital/register-trainee-teachers/pull/4734/files#diff-2e324da006cb703e645d47853fb95bcb14e51ee36f696dd9513e782525ea1eaeR27) changed this won't work as the `ImportRows::call` expects the keyword argument `id`